app/styles/ui/check-runs/_ci-check-run-list-item.scss

```diff
@@ -32,10 +36,14 @@
   }
 
   .view-on-github {
-    display: none;
-    padding: var(--spacing);
-    padding-bottom: var(--spacing-half);
+    margin: var(--spacing);
     color: var(--text-secondary-color);
+    min-height: var(--spacing);
+    min-width: var(--spacing-double);
```

Comment:
It feels odd to use variables that are related to spacing to calculate height and width in situations when they're not directly related to spacing 😕 

I tried out removing these two lines and I personally liked the result: the title would expand to use all the width available and then, when you hover over the item (or its logs), the icon would show up:

https://user-images.githubusercontent.com/1083228/140714396-89342ae9-e36c-4bbf-99c2-e54689dbbb0e.mov

What do you think?